.gitignore

```diff
@@ -6,4 +6,4 @@ yarn-error.log
 dev
 firebase-debug.log
 dev-mantic-markets-firebase-adminsdk*.json
-mantic-markets-firebase-adminsdk*.json
\ No newline at end of file
+mantic-markets-firebase-adminsdk*.jsonbackend/scripts/fetch-sports.ts
```

Comment:
Missing a newline, but why would this be in the gitignore?

Reply:
Oh I'll push a commit to delete this change, this was part of my failed attempt of not leaking the API key lol